source/blog/2018-02-01-firewalld-in-atomic-host.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Firewalld in Atomic Host'
+author: dustymabe
+date: 2018-02-01 18:00:00 UTC
+tags: fedora, atomic
+published: true
+---
+
+# TL;DR
+
+Fedora Atomic Host (and derivatives) will now include the `firewalld`
+package in the base OSTree that is tested, delivered, and released
+every two weeks. Existing users should observe no change as it won't
+be enabled by default.
+
+# Firewalld in Atomic Host
+
+In the past we have had requests to have `firewalld` in Atomic Host
+to enable a better interface into firewall management for
+administrators and management software. It turns out that if you have
+lots of rules to manage, or even multiple pieces of software trying to
+manage different sets of rules on a single system, then `iptables`
+becomes a limitation pretty quickly.
+
+For a while now Atomic Host users have had the ability to package layer
+`firewalld` into Atomic Host, but package layering requires a reboot,
+unless you're using the experimental [livefs](https://www.projectatomic.io/blog/2017/06/rpm-ostree-v2017.6-released/)
+feature. Unfortunately, in certain environments a reboot is not desirable so
+the [Fedora Atomic Working Group](https://pagure.io/atomic-wg)
+recently [approved](https://pagure.io/atomic-wg/issue/372)
+adding `firewalld` to the base OSTree.  `firewalld` will be shipped
```

Comment:
It's not *just* about that issue right?  It's more that we assume more people will use it and hence it makes sense to be included.

Reply:
I guess. I did want to link to the issue to show we had discussed it in the WG though. Want to suggest some text to add that also indicates we think more people will use it?

Reply:
"While Atomic Host supports package layering, live changes are currently experimental.  `firewalld` is also part of the system bootstrap path, and installing it requires restarting docker too.  Because there are a lot of use cases that want it, and it's not too small, we are taking the path of including it in the base Atomic Host."

?